src/helpers/featureFlags.ts

```diff
@@ -21,11 +22,12 @@ export function setConfigFromFeatureFlags(config: IChartConfig, featureFlags: IF
     }
 
     let result = config;
-
-    if (featureFlags.disableKpiDashboardHeadlineUnderline === true) {
+    if (
+        featureFlags.disableKpiDashboardHeadlineUnderline === true &&
+        (!config || isNil(config.disableDrillUnderline))
```

Comment:
What if `config.disableDrillUnderline === false`? I believe the isNil only tests for null and undefined, right?

Reply:
If `config.disableDrillUnderline === false`, isNil will return false, and yes isNil only tests for null and undefined.

Reply:
Right, and thus it will not change the disable prop. Thank you for the clarification :)